editors/code/package.json

```diff
@@ -1,16 +1,28 @@
 {
 	"name": "air",
 	"displayName": "Air",
-	"description": "Air Language Server",
+	"description": "R formatter and language server for Visual Studio Code",
 	"version": "0.0.1",
 	"publisher": "Posit",
 	"license": "MIT",
-	"repository": "https://github.com/posit-dev/air",
+	"homepage": "https://github.com/posit-dev/air",
```

Comment:
What would you think about using the docs site URL here instead?

Reply:
Good idea!